infrastructure/prime-app-template.yml

```diff
@@ -177,8 +185,16 @@ objects:
               - containerPort: 8890
                 protocol: TCP
             volumeMounts:
-              - name: nginx-logs
-                mountPath: /var/log/nginx/
+            - name: vanity-tls-certs
```

Comment:
Im surprised that the certs was missing?

Reply:
We load them directly into Route in prod and test. but not dev.